sql/connect/common/src/main/protobuf/spark/connect/ml.proto

```diff
@@ -84,7 +85,7 @@ message MlCommand {
 message MlCommandResult {
   oneof result_type {
     // The result of the attribute
-    Param param = 1;
+    Expression param = 1;
```

Comment:
I think the unification to Expression is the right direction, but we don't support all Expressions, but only Expression.Literal, I would prefer changing it to Literal here to be explicit.

Reply:
E.g. we cant use `unresolved_function_call` here :)

Reply:
make sense, let me narrow it to `Literal`